.gitignore

```diff
@@ -2,6 +2,7 @@ dist
 yarn-error.log
 node_modules
 coverage
+.DS_store
```

Comment:
Also had PRs being sent on many repos to add node_modules beside all my reluctantly to accept them I did so we could do some actual coding :D